cpu/cc26x0/periph/rfc.c

```diff
@@ -58,31 +58,29 @@ void rfc_irq_disable(void)
     NVIC_DisableIRQ(RF_HW_IRQN);
 }
 
-uint32_t rfc_send_cmd(void *ropCmd)
+uint32_t rfc_send_cmd(uint32_t ropAddr)
 {
-    RFC_DBELL->CMDR = (uint32_t) ropCmd;
+    RFC_DBELL->CMDR = ropAddr;
 
     /* wait for cmd ack (rop cmd was submitted successfully) */
-    while (RFC_DBELL->RFACKIFG << 31);
+    while (RFC_DBELL->RFACKIFG<<31);
```

Comment:
the RIOT coding style doesn't say anything about spaces around operators but most of the existing code seems to have them.
at least put that in a separate commit